src/components/UpdateAppModal/BaseUpdateAppModal.js

```diff
@@ -33,31 +35,42 @@ class BaseUpdateAppModal extends PureComponent {
                     onSubmit={this.submitAndClose}
                     onClose={() => this.setState({isModalOpen: false})}
                     isVisible={this.state.isModalOpen}
+                    type={this.props.isSmallScreenWidth
+                        ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED : CONST.MODAL.MODAL_TYPE.CONFIRM}
                 >
-                    <HeaderWithCloseButton
-                        title="Update App"
-                        onCloseButtonPress={() => this.setState({isModalOpen: false})}
-                    />
-                    <Text style={[styles.textLabel, styles.p4]}>
-                        A new version of Expensify.cash is available.
-                        Update now or restart the app at a later time to download the latest changes.
-                    </Text>
-                    {this.props.onSubmit && (
+                    <View style={[styles.m5]}>
```

Comment:
No need for square brackets if it's just a single style.

```suggestion
                    <View style={styles.m5}>
```